### PR TITLE
Add social share buttons

### PR DESCRIPTION
**Pull request by emranbm**
Add LinkedIn, Twitter, and Reddit share buttons.

### CODE REVIEW

**Comment by emranbm** [comment]
It's gonna look like this:
Screenshot from 2022-02-09 19-59-03

File: README.md [hunk 1]
```diff
@@ -35,7 +35,17 @@ A very common misconception is that quality takes time and money. Actually, anyt
 ## Maintenance and Contribution
 QMM is an [open-source project](https://github.com/qualiticcommunity/qualiticcommunity.github.io) and its maintainers highly try to avoid any exercise of authority, while keeping it usable, maintainable, and practical.  
 So contributions of any kind to the maturity model - through opening issues, pull requests, or contacting us any other way - are appreciated.  
-Feel free to let us know how your experience using this maturity model was, and what can make it better for your team/organization.
+
+If you like QMM, please share it! And let us know how your experience using this maturity model was, and what can make it better for your team/organization.
```
Comment (mojyou) [inline README.md:39]: I suggest this:
If you like QMM, please support us by sharing it.

**Review by mojyou — COMMENTED** [review]
There is just one suggestion, which is not compulsory.
Thanks.

File: README.md (hunk 1 above)
Comment (emranbm) [reply README.md]: I applied your suggestion but kept the next sentence too. Please review it again. thanks.

**Review by mojyou — APPROVED** [review]
Thanks Emran.

**Pull request merged**